### DISCUSSION TITLE
where to find the latest code?

### DISCUSSION THREAD

**Discussion by freemant2000** [elimintz/justpy#573] [General]
Hi,

This may seem a silly question: Where is the latest code? From pip I can get v0.9.11 which contains modules like justpy.core, but the github repository here doesn't seem to have it?

Thanks in advance

**Comment by WolfgangFahl** [comment]
@freemant2000  IMHO there is no such thing as a silly question
For example:
https://github.com/justpy-org/justpy/tree/master/jpcore
has the jpcore code.

The repository has recently moved but the links from

https://justpy.io/
and

https://stackoverflow.com/tags/justpy/info

and the pypi backlinks from all releases:
https://pypi.org/search/?q=justpy

should all be correct for releases of recent milestones.
